src/routes/post.js

```diff
@@ -110,6 +110,14 @@ router.post('/:id/like', loginRequired, checkIdParams, async (ctx) => {
   })
 })
 
+router.post('/:id/react', loginRequired, checkIdParams, async (ctx) => {
+  ctx.body = await Post.react(ctx.identity.id, ctx.request.body.emoji, ctx.params.id)
+})
+
```

Comment:
Correct me if I'm wrong but we can have duplicate reacts and with the same emojie.

Reply:
Everything else LGTM 👍 
Well Done ✅